### DISCUSSION TITLE
error slither: command not found.

### DISCUSSION THREAD

**Discussion by kartik-giri** [crytic/slither#2356] [Installation]
Hey guyz i am trying to run slither but getting this error slither: command not found. Even though I have installed it using pip. I would appreciate the help thanks.

Errror :- slither: command not found

**Comment by elopez** [comment]
Hi @kartik-giri ! It's possible you need to configure the $PATH variable on your system so that Python programs are available on your shell. On Windows there is a checkbox that you need to check during the Python installation.

In the meantime, until you sort that out on your computer, you can invoke slither with `python3 -m slither` instead of `slither`
